README.md

```diff
@@ -33,7 +44,7 @@ as a Nuget package or you can download the source using the instructions above.
 
 ### Prerequisites
```

Comment:
The above may want to be revamped to mention Windows Universal & WP 8.1 0r made more generic?